src/test/java/edu/cornell/kfs/rass/batch/xml/fixture/RassXmlAwardEntryFixture.java

```diff
@@ -0,0 +1,104 @@
+package edu.cornell.kfs.rass.batch.xml.fixture;
+
+import java.util.Date;
+
+import org.apache.commons.lang3.StringUtils;
+import org.joda.time.format.DateTimeFormatter;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.rass.batch.xml.RassXmlAwardEntry;
+import edu.cornell.kfs.rass.batch.xml.RassXmlDocumentWrapperMarshalTest;
+
+public enum RassXmlAwardEntryFixture {
+    FIRST("141414", "OS", "2345", "First Example Project", "2017-12-31", null, new KualiDecimal(5300000.000), new KualiDecimal(700000.000), new KualiDecimal(6000000.000),
+            StringUtils.EMPTY, StringUtils.EMPTY, StringUtils.EMPTY, StringUtils.EMPTY, StringUtils.EMPTY, "3434", "Y", null,
+            piFixtures(RassXMLAwardPiCoPiEntryFixture.cah292_PRIMARY)),
+    ANOTHER("141415", "RS", "24680", "Another Example", null, "2050-12-31", new KualiDecimal(0), new KualiDecimal(0), new KualiDecimal(0),
+            StringUtils.EMPTY, "GRT", StringUtils.EMPTY, StringUtils.EMPTY, StringUtils.EMPTY, "2374", "Y", "2022-09-30",
+            piFixtures(RassXMLAwardPiCoPiEntryFixture.NO_NAME_PRIMARH)),
+    NULL_AMOUNTS("35656", "RS", "24680", "Award with empty totals", null, "2050-12-31", new KualiDecimal(0), null, null,
+            StringUtils.EMPTY, "GRT", StringUtils.EMPTY, StringUtils.EMPTY, StringUtils.EMPTY, "2374", "Y", "2022-09-30",
+            piFixtures(RassXMLAwardPiCoPiEntryFixture.jdh34_CO_PI));
+    public final String proposalNumber;
+    public final String status;
+    public final String agencyNumber;
+    public final String projectTitle;
+    public final Date startDate;
```

Comment:
java.util.Date objects are mutable, so you may want to use Joda DateTime objects on this enum instead. If you can use the appropriate static method on the related XML adapter class, then you can easily create the needed DateTime objects while still minimizing duplicate code.

Reply:
changed